etc/geoserver/gwc/geowebcache.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<gwcConfiguration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xmlns="http://geowebcache.org/schema/1.8.0"
+  xsi:schemaLocation="http://geowebcache.org/schema/1.8.0 http://geowebcache.org/schema/1.8.0/geowebcache.xsd">
+  <version>1.8.0</version>
+  <backendTimeout>120</backendTimeout>
+  <serviceInformation>
+    <title>MobiData BW GeoWebCache</title>
+    <description>GeoWebCache is an advanced tile cache for WMS servers. It supports a large variety of protocols and
```

Comment:
This could mention that the tile layers are for DE-BW, provided by MobiData BW, run by NVBW.

```suggestion
    <description>Map tile service for Baden-Württemberg (DE-BW), part of MobiData BW, provided by NVBW.
```